cluster-scope/bundles/b4mad-racing/kustomization.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base/core/namespaces/b4mad-racing
```

Comment:
We shouldn't create single-resource bundles. This should just be included directly from the `kustomization.yaml`.

Reply:
oops! Thanks for the feedback, I have updated the PR